pkg/xds/topology/outbound.go

```diff
@@ -469,6 +469,9 @@ func createMeshExternalServiceEndpoint(
 	tls := mes.Spec.Tls
 	meshName := mesh.GetMeta().GetName()
 	service := mes.Meta.GetName()
+	if tls == nil {
+		return nil
```

Comment:
probably we shouldn't return when tls is nil just don't configure it

Reply:
this is just temporary so my test doesn't NPE :D